README.rst

```diff
@@ -630,6 +634,27 @@ This is achieved by setting the ``FFLAGS`` variable during LoopTools configurati
 
     FFLAGS=-fPIC ./configure
 
+COLLIER support
+---------------
+
+It is possible to use COLLIER_ for calculating the loop functions,
+instead of using SOFTSUSY's loop functions.  To enable COLLIER
+configure FlexibleSUSY via ::
+
+   ./configure --with-loop-libraries=collier
+
+To use the COLLIER library and header files from a specific
+directory configure via ::
+
+    COLLIER_DIR=/path/to/COLLIER-x.y.z
+
+    ./configure --with-loop-libraries=collier \
+       --with-collier-incdir=$COLLIER_DIR/modules \
+       --with-collier-libdir=$COLLIER_DIR
+
+Note: versions since COLLIER-1.2.3 were tested so far.
+Also, COLLIER static library should be configured with ``-Dstatic=ON`` option
+and compiled with ``Fortran_FLAGS = -fPIC`` of ``COLLIER-x.y.z/build/CMakeFiles/collier.dir/flags.make`` file.
```

Comment:
I think that `fPIC` can be simply added from command line, as
```
-DCMAKE_Fortran_FLAGS=-fPIC
```

Reply:
thank you, I didn't know it. I will check it.

Reply:
@wkotlarski, does this command work for you? For me - no. The problem could be that `CMAKE_Fortran_FLAGS` is not an `option` neither `CACHE` variable.

Reply:
Can you check this
```
-DCMAKE_POSITION_INDEPENDENT_CODE=ON
```

Reply:
Instruction updated in cd4280e.